src/test/java/uk/gov/justice/hmpps/prison/util/builders/OffenderProfileDetailsBuilder.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.hmpps.prison.util.builders
+
+import uk.gov.justice.hmpps.prison.service.DataLoaderRepository
+
+enum class ProfileType {
+  YOUTH
+}
+
+class OffenderProfileDetailsBuilder(
+  private val code: String,
+  private val type: ProfileType,
+) {
+
+  // use SQL to save because the JPA mapping may not be correct
+  fun save(
```

Comment:
we could use the JPA repo to do this, but SQL is more reliable exactly because there is issue to be fixed in the JPA, eventually we will relay on either JPA or the service for creating the test data, but for now use good old fashioned SQL